src/index.html

```diff
@@ -11,7 +11,136 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label" for="surname">
+          Surname:
+          <input 
+            name="txt-surname" 
+            type="text" 
+            id="surname" 
+            autocomplete="off" 
+            minlength="4" 
+            maxlength="24"
+          >
+        </label>
+        <label class="label" for="name">
+          Name:
+          <input 
+            name="txt-name" 
+            type="text" 
+            id="name" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="label" for="age">How old are You?
+          <input name="txt-age" id="age" 
+            type="number" min="1" 
+            max="100" value="12"
+          >
```

Comment:
```suggestion
          <input
            name="txt-age"
            id="age" 
            type="number"
            min="1" 
            max="100"
            value="12"
          >
```
1 attribute per line if there are more then 2
[point 3
](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Reply:
Done!